packages/market-service/src/foxy/foxy.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  HistoryData,
+  HistoryTimeframe,
+  MarketCapResult,
+  MarketData,
+  MarketDataArgs,
+  PriceHistoryArgs
+} from '@shapeshiftoss/types'
+import dayjs from 'dayjs'
+
+import { MarketService } from '../api'
+import { CoinCapMarketCap } from '../coincap/coincap-types'
+import { RATE_LIMIT_THRESHOLDS_PER_MINUTE } from '../config'
+import { bn } from '../utils/bignumber'
+import { isValidDate } from '../utils/isValidDate'
+import { rateLimitedAxios } from '../utils/rateLimiters'
+
+export const FOXY_CAIP19 = 'eip155:1/erc20:0x61fcabb591d63d00e897a67c64658d376fead816'
+const FOX_COINCAP_ID = 'fox-token'
+
+const axios = rateLimitedAxios(RATE_LIMIT_THRESHOLDS_PER_MINUTE.COINGECKO)
+
+export class FoxyMarketService implements MarketService {
```

Comment:
wait, why do we even need this, if coincap is the adapter backing this?

just update the coincap caip adapters to point the foxy caip19 at the asset id for the underlying token?

Reply:
as discussed:

this is valid over just the caip adapter idea, please comment heavily that this is only a stop gap until we get coingecko/coincap upstream sources, at which point this is to be deleted.